static/js/components/email/EmailCompositionDialog.js

```diff
@@ -86,6 +87,42 @@ export default class EmailCompositionDialog extends React.Component {
     }
```

Comment:
Can you remove this function? `this.state.editorState` shouldn't ever be null or undefined here so we shouldn't need to do this since `this.state` is set in the constructor and `editorStateFromProps` always returns sets some value for `editorState`